features/plain_features/autonotify.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Using auto notify
+
+Background:
+  Given I set environment variable "API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
+  And I configure the bugsnag endpoint
+  And I have built the service "app"
+
+# Scenario: An error report is sent when an unhandled crash occurs
```

Comment:
Is it worth just removing these scenarios for the time being, are they valuable at all as placeholders?

Reply:
Deleted